common/meta/grpc/src/kv_api_impl.rs

```diff
@@ -19,12 +19,12 @@ use common_meta_types::PrefixListReply;
 use common_meta_types::UpsertKVAction;
 use common_meta_types::UpsertKVActionReply;
 
-use crate::grpc::grpc_action::GetKVAction;
-use crate::grpc::grpc_action::MGetKVAction;
-use crate::grpc::grpc_action::PrefixListReq;
+use crate::grpc_action::GetKVAction;
+use crate::grpc_action::MGetKVAction;
+use crate::grpc_action::PrefixListReq;
 use crate::MetaGrpcClient;
 
-#[async_trait::async_trait]
+#[tonic::async_trait]
```

Comment:
Why is it switched from `async_trait` to `tonic`?

Reply:
`tonic::async_trait` is just re-pub `async_trait::async_trait`, using `tonic` reduce a dep crate:)

Reply:
make sense 🤔